lib/AST/Decl.cpp

```diff
@@ -11454,25 +11457,33 @@ DestructorDecl *DestructorDecl::getSuperDeinit() const {
   return nullptr;
 }
 
+SourceLoc FuncDecl::getStartLoc() const {
+  auto startLoc = StaticLoc;
```

Comment:
Any reason not to invert this, ie. isValid + early return? This way is technically shorter I suppose, but also makes me do a double take 😅

Reply:
Really I was just matching the style of `getSourceRange`, happy to change

Reply:
Updated to use early return for both